xbmc/video/ContextMenus.cpp

```diff
@@ -194,7 +195,17 @@ void SetPathAndPlay(CFileItem& item, const std::string& player)
     const ContentUtils::PlayMode mode = item.GetProperty("CheckAutoPlayNextItem").asBoolean()
                                             ? ContentUtils::PlayMode::CHECK_AUTO_PLAY_NEXT_ITEM
                                             : ContentUtils::PlayMode::PLAY_ONLY_THIS;
-    VIDEO_UTILS::PlayItem(std::make_shared<CFileItem>(item), player, mode);
+
+    if (item.GetStartOffset() != STARTOFFSET_RESUME &&
+        item.GetVideoInfoTag()->m_type == MediaTypeMovie)
```

Comment:
Could you please add a `//! @todo get rid of special handling for movie versions`, so we can pick this up in a follow-up PR?